README.md

```diff
@@ -46,25 +65,31 @@ brew install poetry
 1. Install Python & Poetry (above)
 2. Clone the [GG2 Github repo](https://github.com/DistributedProofreaders/guiguts-py)
    or a fork thereof.
-3. In the cloned GG2 directory, create a virtual environment using the version of
+3. In the cloned GG2 directory, create a virtual environment using the version(s) of
```

Comment:
The virtual environment still only uses one python version. Or perhaps `using one of the versions` or `using a version of`?

Reply:
Yes, that's clearer - thanks